README.md

```diff
@@ -24,6 +25,7 @@ nice visual feedback.
 
 * Vim with `+terminal`, or Neovim >= 0.5
 * Under Windows, only Neovim is supported at the moment
+* Target completion requires CMake 3.14 or later
```

Comment:
By the way, we should add this line to doc/cmake.txt as well

Reply:
Oops. I missed this section is in docs as well. Will add.

Reply:
Should I leave [this](https://github.com/cdelledonne/vim-cmake/pull/64/files#diff-b73fc732b2dcef182f1fed830367b06d323bb053305436c27b6c6321d82862b3R140-R141) in then?

Reply:
I would leave this in, yes (just add a blank line right after these two lines, forgot to point it out earlier)